### PR TITLE
[12.0][MIG] mrp_bom_note

### PR DESCRIPTION
**Pull request by lbelorgey**
Standard migration of module mrp_bom_note.

### CODE REVIEW

**Comment by elvise** [comment]
Hi, any good news for merge...?
If I can help please let me know...

File: mrp_bom_note/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Ana Juaristi <anajuaristi@avanzosc.es>
+* Bima Wijaya <bimajatiwijaya@gmail.com>
+* Agung Rachmatullah <agung.rachmatullah@ymail.com>
```
Comment (ps-tubtim) [inline mrp_bom_note/readme/CONTRIBUTORS.rst:5]: Please add you as contributor.

**Review by marylla — APPROVED** [review]
Hi there,
I made a functional review. LGTM. :)

File: mrp_bom_note/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (lbelorgey) [reply mrp_bom_note/readme/CONTRIBUTORS.rst:5]: Done but the acute accent in my name seems to be problematic in the generated file index.html.

**Comment by LoisRForgeFlow** [comment]
@lbelorgey Please squash your migration commits into one.

**Review by ps-tubtim — APPROVED** [review]

**Review by AdriaGForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**